ads/opctl/operator/lowcode/forecast/utils.py

```
@@ -70,8 +72,7 @@ def _build_metrics_per_horizon(
         Dataframe with Mean sMAPE, Median sMAPE, Mean MAPE, Median MAPE, Mean wMAPE, Median wMAPE values for each horizon
     """
     actuals_df = data[target_columns]
-    forecasts_df = pd.concat([df[target_col] for df in outputs], axis=1)
-
+    forecasts_df = pd.concat([df[target_col].iloc[-horizon:] for df in outputs], axis=1)
```

Can we add a check that the `df[target_col]` is of at least length horizon? If not, let's throw an error that @prasankh built